src/ui/public/kuery/lib/filter_migration/lib/__tests__/exists.js

```diff
@@ -0,0 +1,35 @@
+import expect from 'expect.js';
+import { convertExistsFilter } from '../exists';
+
+describe('filter to kuery migration', function () {
+
+  describe('exists filter', function () {
+
+    it('should return a kuery node equivalent to the given filter', function () {
+      const filter = {
+        meta: {
+          type: 'exists',
+          key: 'foo',
+        }
+      };
+      const result = convertExistsFilter(filter);
+
+      expect(result).to.have.property('type', 'function');
+      expect(result).to.have.property('function', 'exists');
+      expect(result.arguments[0].value).to.be('foo');
+    });
+
+    it('should return undefined if the given filter is not of type "exists"', function () {
```

Comment:
Returning undefined and throwing an exception for a non-exceptional scenario both seem equally hacky to me. Since these are private functions used only by `filterToKueryAST`, it doesn't seem terribly important, does it? 

If you're strongly against returning `undefined`, perhaps I should add a `supports` function to each converter module that returns true/false depending on whether that module supports the given filter?